snaps/reference/snaps-api.md

```diff
@@ -874,6 +876,10 @@ An object containing the contents of the notification:
   We recommend using `type: "inApp"` because there's no guarantee that native notifications are
   displayed to the user.
 - `message` - A message to show in the notification.
+- `title` (optional) - The title of the expanded view, shown when a user expands the notification.
```

Comment:
Perhaps we can say, "The title of the expanded view, shown when a user expands the notification. This parameter is required if you want to display the expanded view."?